libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -2285,6 +2301,11 @@ void IRGeneratorForStatements::endVisit(Identifier const& _identifier)
 	{
 		// no-op
 	}
+	else if (dynamic_cast<ErrorDefinition const*>(declaration))
+	{
+		// TODO should this rather be an assertion? Can we get here?
+		// no-op
+	}
```

Comment:
I think that we can unless you specifically did something to prevent revert arguments from being visited.

Putting an assert here and running tests should give us the answer.

Reply:
This is all fixed when it in implemented, I think.